sql/core/src/main/scala/org/apache/spark/sql/execution/ExternalAppendOnlyUnsafeRowArray.scala

```diff
@@ -82,6 +82,8 @@ private[sql] class ExternalAppendOnlyUnsafeRowArray(
 
   private var numFieldsPerRow = 0
 
+  private var numRowBufferedInMemory = 0
```

Comment:
We need this variable? Could we use `inMemoryBuffer.length` instead?

Reply:
Make sense. Done. Thank you